src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -738,7 +738,6 @@ class DocBaseViewer extends BaseViewer {
             disableFontFace,
             disableRange,
             disableStream,
-            isEvalSupported: false,
```

Comment:
This was a workaround for [this](https://github.com/advisories/GHSA-wgrm-67xf-hhpq) security issue which is patched in version 4.2.67.